File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>

using namespace std;

string solution(string s) 
{
    vector<int> vec;
    string tmp = "";
    for(char ch : s)
    {
        if(ch != ' ')
        {
            tmp += ch;
        }
        else if (ch == ' ')
        {
            vec.push_back(stoi(tmp));
            tmp = "";
        }
    }
    vec.push_back(stoi(tmp));
    sort(vec.begin(), vec.end());
    string result = to_string(vec[0]) + " " + to_string(vec[vec.size() - 1]);
    return result;
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cs
using System;

public class Solution 
{
    public string solution(string s) 
    {
        string[] str = s.Split(' ');
        int[] arr = new int [str.Length];
        for(int i=0; i<str.Length; i++)
        {
            arr[i] = Int32.Parse(str[i]);
        }
        Array.Sort(arr);
        string ans = arr[0].ToString() + " " + arr[arr.Length - 1].ToString();
        return ans;
    }
}